modules/ROOT/partials/integrations/svelte-tech-ref.adoc

```diff
@@ -66,9 +68,9 @@ Specifies the {cloudname} channel to use. For information on {cloudname} deploym
 
 Specified an Id for the editor. Used for retrieving the editor instance using the `+tinymce.get('ID')+` method.
 
-*Default value:* Automatically generated UUID
+*Type:* `+String+`
 
-*Type:* String
+*Default value:* Automatically generated UUID
```

Comment:
As above, if we care to add the link?

Reply:
Fixed and also cleaned up some other minor inconsistencies.